### ISSUE TITLE
`molecule status` raises IndexError when container is stopped

### ISSUE THREAD

**Issue by keyboardscience** [metacloud/molecule#466]
`molecule status` raises an uncaught `IndexError` when container is stopped.

 Fix entails adding `all=True` to the `self._docker.containers(filters={'name': name})...` call in `molecule/drivers/dockerdriver.py:176`.

I'm working on the PR to fix, but I could some advice on how best to test. Here is what I have so far:
```
def test_status_when_stopped(docker_instance):
    ''' Test docker_instance.status when test subject -- container -- is stopped or exited '''
    docker_instance.up()
    docker_instance._docker.stop(container='test1')

    assert 'Exited' in docker_instance.status()[0].state
```

To reproduce:
```
> docker stop test-01 && molecule -v && molecule status
1.11.0
Traceback (most recent call last):
  File "/usr/local/bin/molecule", line 11, in <module>
    sys.exit(main())
  File "/Library/Python/2.7/site-packages/molecule/cli.py", line 68, in main
    CLI().main()
  File "/Library/Python/2.7/site-packages/molecule/cli.py", line 64, in main
    util.sysexit(c.execute()[0])
  File "/Library/Python/2.7/site-packages/molecule/command/status.py", line 57, in execute
    status = self.molecule.driver.status()
  File "/Library/Python/2.7/site-packages/molecule/driver/dockerdriver.py", line 176, in status
    cd = self._docker.containers(filters={'name': name})[0]
IndexError: list index out of range
```

**Comment by retr0h** [comment]
> Fix entails adding all=True to the self._docker.containers(filters={'name': name})... call in molecule/drivers/dockerdriver.py:176.

Yeah, the problem with that, is the way in which docker reports status.  If you add that, then `molecule status` when the container is offline, will not show the custom offline message, but instead show the number of minutes since the container was stopped.

**Comment by retr0h** [comment]
The use case is displaying the default docker status while machine is up, and `not_created` when down.  Looks like I introduced that bug when implementing this logic.

**Comment by keyboardscience** [comment]
@retr0h Is molecule's preference to display a more user friendly message? Also, I don't quite understand what you were getting at when you say, "when the container is offline, will not show the custom offline message, but instead show the number of minutes since the container was stopped.". Could you clarify?

With the proposed change, the status describes how much time has elapsed since the container 'Exited'. If the container does not exist, the `not_created` state, the status reports 'Not Created'.

**Comment by retr0h** [comment]
> With the proposed change, the status describes how much time has elapsed since the container 'Exited'. If the container does not exist, the not_created state, the status reports 'Not Created'.

That should be fine.

**Comment by retr0h** [comment]
Released molecule 1.11.3 to pypi.
